src/components/events/PointsDisplay/style.module.scss

```
@@ -0,0 +1,33 @@
+.circle {
+  align-items: center;
+  border-radius: 50%;
+  display: flex;
+  height: 50px;
+  justify-content: center;
+  position: absolute;
+  right: 2.5%;
+  top: 5%;
+  width: 50px;
+  z-index: 1;
+
+  &.blue {
+    background-color: var(--theme-primary-4);
+  }
```

```suggestion
  background-color: var(--theme-primary-4);
```


you don't need a class name for blue since this is essentially the default background color for all .circle elements - i would only keep the green one as a more specific override but also rename it for specificity to attended or something better. i agree w sean that we shouldn't be using colors to indicate attendance anyways